README.md

```diff
@@ -115,8 +115,10 @@ JDBC_MAX_ACTIVE 			| The maximum number of active connections that can be alloca
 JDBC_MIN_IDLE 				| The minimum number of established connections that should be kept in the pool at all times. | `3`
 JDBC_MAX_IDLE 				| The maximum number of connections that should be kept in the pool at all times. | `25`
 JDBC_MAX_WAIT 				| The maximum number of milliseconds that the pool will wait (when there are no available connections) for a connection to be returned before throwing an exception. | `30000`
-JDBC_INITIAL_SIZE 			| The initial number of connections that are created when the pool is started. | `4`
-JDBC_CONNECTION_PROPERTIES 	| The connection properties that will be sent to our JDBC driver when establishing new connections. Format of the string must be `[propertyName=property;]*`  | 
+JDBC_INITIAL_SIZE 			| The initial number of connections that are created when the pool is started. | `3`
+JDBC_CONNECTION_PROPERTIES 	| The connection properties that will be sent to our JDBC driver when establishing new connections. Format of the string must be `[propertyName=property;]*`  |
```

Comment:
slight tweaks to wording requested since most of these are specific to the connection pool settings>>
The initial number of database connections that are created when the pool is started.
The database connection pool properties that deploying sends to the JDBC driver when creating new database connections. Format of the string must be `[propertyName=property;]*`

The number of milliseconds that an object is allowed to sit idle in the database connection pool before it is eligible for eviction.

Also for line 117>>The number of milliseconds that the database connection pool will wait (when there are no available connections) for a connection to be returned before throwing an exception.